README.md

```diff
@@ -1 +1,41 @@
-# To-Do-List
\ No newline at end of file
+# To-Do-List
+
+> The project focused on creating To do list structure
+
+## Built With
+- HTML
+- CSS
+-JAVASCRIPT
+​
+## Live Demo (if available)
+
+- https://alphantihinduka.github.io/Alpha-Mobile-portifolio/
+
+## Getting Started
+
+-Install webpack
+
+-Used the microverse linter documentation
+
+
+-set up linter for HTML and CSS
+
+-Install eslintrc.json.
```

Comment:
- [ ] (Optional) It would be nice to include instructions that direct people on how to clone the application and install the dependencies. Something like this:

```
git clone repo URL
cd <Repo Folder Name>
npm install
```